src/cf-vars.less

```diff
@@ -44,15 +47,18 @@
     - cf-core
 */
 
-// Sizing variables 
+// Sizing variables
 
 @base-font-size-px:             16px;
 @base-line-height-px:           22px;
 @base-line-height:              unit(@base-line-height-px / @base-font-size-px);
 @mobile-max:                    599px;
 @tablet-min:                    600px;
+@tablet-max:                    800px;
+@desktop-min:                   801px;
+@wall-min:                      1200px;
```

Comment:
Mmm... that's kind of a different concept than the device-type concept of the other ones. Maybe call it `@tv-min`?

Also, if that's the case, shouldn't it be more like 1920px?